### ISSUE TITLE
Large stack error #35563

### ISSUE THREAD

**Issue by jim4067** [anza-xyz/agave#1459]
#### Problem
Getting a stack error, when building with the  1.18.8 >

```bash
Error: Function _ZN112_$LT$solana_program..instruction..InstructionError$u20$as$u20$solana_frozen_abi..abi_example..AbiEnumVisitor$GT$13visit_for_abi17h14a8348cc8de07b8E Stack offset of 4608 exceeded max offset of 4096 by 512 bytes, please minimize large stack variables
```

I don't think the issues lies my program having large accounts because after downgrading to 1.17.28 before worked.

#### Environment
Solana `solana-cli 1.18.14 (src:d8e08244; feat:4215500110, client:SolanaLabs)`

**Comment by LucasSte** [comment]
Would you mind sharing a reproducible example?

**Comment by joncinque** [comment]
This was fixed with https://github.com/solana-labs/solana/pull/35045, but it's part of 1.18.2 -- be sure to use at least that version of `solana-program`

**Comment by jim4067** [comment]
Thanks.  This worked.

> This was fixed with [solana-labs#35045](https://github.com/solana-labs/solana/pull/35045), but it's part of 1.18.2 -- be sure to use at least that version of `solana-program`
